src/test/java/teammates/test/cases/BaseComponentTestCase.java

```diff
@@ -97,7 +97,8 @@ protected StudentAttributes getStudent(StudentAttributes student) {
     }
 
     protected void removeAndRestoreTypicalDataBundle() {
-        removeAndRestoreDataBundle(typicalBundle);
+        DataBundle dataBundle = getTypicalDataBundle();
+        removeAndRestoreDataBundle(dataBundle);
```

Comment:
I don't know exactly but we not need to reload data as some test are depend on previous test data so  ` removeAndRestoreDataBundle(typicalBundle);`  is not working.
Also `removeAndRestoreTypicalDataBundle()` is used in different other test packages as well so would it be good to change this method ?